fleetspeak/src/server/components/prometheus/prometheus.go

```diff
@@ -186,15 +187,30 @@ var (
 	)
 )
 
+// Returns a well-defined string representing labels from a given client.
+func clientLabelsFromClientData(cd *db.ClientData) string {
```

Comment:
If there's no other `clientLabels` function, then I suggest removing the `FromClientData` suffix. It should be redundant when you read the call `clientLabels(clientData)` would already convey that you're getting it from the client data. If you can get it from different types, since there's no overload, then keep the `from..` suffix
https://google.github.io/styleguide/go/decisions.html#external-context-vs-local-names